commands/team.go

```diff
@@ -222,6 +233,43 @@ func deleteTeam(c client.Client, team *model.Team) (bool, *model.Response) {
 	return c.PermanentDeleteTeam(team.Id)
 }
 
+func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("Not enough arguments.")
+	}
+
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+	if !confirmFlag {
+		var confirm string
+		fmt.Println("Have you performed a database backup? (YES/NO): ")
+		fmt.Scanln(&confirm)
+
+		if confirm != "YES" {
+			return errors.New("ABORTED: You did not answer YES exactly, in all capitals.")
+		}
+		fmt.Println("Are you sure you want to archive the teams specified?  All data will be deleted? (YES/NO): ")
```

Comment:
As the team is being archived, we can lower down a little the error message:

```suggestion
		fmt.Println("Are you sure you want to archive the specified teams? (YES/NO): ")
```

Reply:
Done.